cmd/cgr-engine/cgr-engine.go

```diff
@@ -439,6 +433,11 @@ func main() {
 	}
 	utils.Logger.SetLogLevel(lgLevel)
 
+	if *printConfig {
+		cfgJSON := utils.ToIJSON(cfg.AsMapInterface(cfg.GeneralCfg().RSRSep))
+		utils.Logger.Info(fmt.Sprintf("Configuration loaded from %s:\n%s", *cfgPath, cfgJSON))
```

Comment:
Please add here the path between <>, something like: Configuration loaded from <%s> ...